scripts/build_arbor.sh

```diff
@@ -22,6 +22,24 @@ if [ ! -f "$arb_checked_flag" ]; then
         git checkout "$ns_arb_branch" >> "$out" 2>&1
         [ $? != 0 ] && exit_on_error "see ${out}"
     fi
+
+    # workaround for 0.2.1 install issue
+    if [ "$ns_arb_branch" == "v0.2.1" ]; then
+        patch -p1 -d "$arb_repo_path" << '__end__'
+--- a/CMakeLists.txt
```

Comment:
Gargh! Can we bump version again?
We could make `libunwind` always of by default?

Reply:
We can certainly bump version again. Bump early, bump often!

Reply:
Thinking also we should have some sort of build test against installed arbor incorporated into the CI script.